### PR TITLE
Configuring ESLint

### PR DESCRIPTION
**Pull request by lucashanke**
This change is [Reviewable](https://reviewable.io/reviews/ayr-ton/kamu/94)

### CODE REVIEW

File: assets/src/App.test.jsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import { shallow } from 'enzyme';
 import Header from './Header';
 import App from './App';
 import { getLoggedUser } from './services/ProfileService';
-import { currentUser } from "../test/userHelper";
+import { currentUser } from '../test/userHelper';
 
 jest.mock('./services/ProfileService');
 
-const createComponent = (props) => shallow(<App {...props} />);
+const createComponent = props => shallow(<App {...props} />);
```
Comment (macecchi) [inline assets/src/App.test.jsx]: maybe standardize so all function arguments have `()`?

Comment (lucashanke) [reply assets/src/App.test.jsx]: Done ;)

File: assets/src/Header.jsx [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
 
 function Header({ borrowedBooksCount, history }) {
-  const redirect = (url) => history.push(url);
+  const redirect = url => history.push(url);
   return (
-    <AppBar	className="header">
+    <AppBar className="header">
       <Toolbar>
-        <a href="#" className="header-content" onClick={(e) => { e.preventDefault(); redirect(getHomeEndpoint()); }}>
+        <a href={getHomeEndpoint()} className="header-content">
```
[the remark this answers is not in the document]
Comment (lucashanke) [reply assets/src/Header.jsx]: You're right. Just fixed it. 😙

File: assets/src/App.jsx [hunk 3]
```diff
@@ -17,29 +17,37 @@ class App extends Component {
...
+          <Header borrowedBooksCount={user ? user.borrowed_books_count : 0} />
+          {user && (
             <div id="content">
-              <Route exact path="/" render={({ history }) => (
-                <LibraryRedirector history={history}>
-                  <LibrarySelector />
-                </LibraryRedirector>
-              )} />
+              <Route
+                exact
```
[the remark this answers is not in the document]
Comment (lucashanke) [reply assets/src/App.jsx]: Not gonna happen :P

[the remark this answers is not in the document]
Comment (macecchi) [reply assets/src/App.jsx]: okay :(

**Pull request merged**